PHC-Android/PHC/src/main/java/phc/android/SearchResultsFragment.java

```diff
@@ -84,6 +104,17 @@ public void onActivityCreated (Bundle savedInstanceState) {
     }
 
     @Override
+    public void onSaveInstanceState(Bundle outState) {
+        // Save the cached results
+        outState.putParcelableArray(CACHED_RESULTS, mSearchResults);
```

Comment:
this is interesting - i thought that Bundles could only store primitive types. so it can store parceled information too? is this because once these things are parceled, they basically become serialized to ints?